common/djangoapps/third_party_auth/tests/specs/test_lti.py

```diff
@@ -121,7 +121,8 @@ def test_reject_initiating_login(self):
     def test_reject_bad_login(self):
         login_response = self.client.post(
             path=LTI_TPA_LOGIN_URL, content_type=FORM_ENCODED,
-            data="invalid=login"
+            data="oauth_consumer_key=12345&oauth_signature_method=HMAC-SHA1&"
+                 "oauth_signature=STPWUouDw%2FlRGD4giWf8lpGTc54%3D&oauth_timestamp=1436823553&oauth_version=1.0"
```

Comment:
Why was this test changed?  Isn't it testing the behavior when an invalid-login is sent?  Is the `data` value being passed in now a valid value?